beta/src/components/MDX/Sandpack/OpenInCodeSandboxButton.tsx

```diff
@@ -3,29 +3,21 @@
  */
 
 import * as React from 'react';
-import {useCodeSandboxLink} from '@codesandbox/sandpack-react';
-import cn from 'classnames';
+import {UnstyledOpenInCodeSandboxButton} from '@codesandbox/sandpack-react';
 import {IconNewPage} from '../../Icon/IconNewPage';
 
-export const OpenInCodeSandboxButton = ({className}: {className?: string}) => {
-  const url = useCodeSandboxLink();
-
+export const OpenInCodeSandboxButton = () => {
   return (
-    <a
-      className={cn(
-        'text-sm text-primary dark:text-primary-dark inline-flex items-center hover:text-link duration-100 ease-in transition mx-1',
-        className
-      )}
-      href={url}
-      rel="noreferrer noopener"
-      target="_blank"
+    <UnstyledOpenInCodeSandboxButton
+      className="text-sm text-primary dark:text-primary-dark inline-flex items-center hover:text-link duration-100 ease-in transition mx-1 ml-3 md:ml-1"
       title="Open in CodeSandbox">
-      <span className="hidden md:inline">
-        <IconNewPage className="inline mb-0.5 text-base" /> Fork
-      </span>
-      <span className="inline md:hidden">
-        <IconNewPage className="inline mb-0.5 text-base" /> Fork
-      </span>
-    </a>
+      <IconNewPage
+        className="inline mb-0.5 ml-1 mr-1 relative"
```

Comment:
```suggestion
        className="inline mb-0.5 ml-1 mr-1 relative top-px"
```

Reply:
Wow, thanks @harish-sethuraman, I spent some time figuring out how I could do this.

Reply:
:)